app/views/application/decision.njk

```diff
@@ -63,7 +63,8 @@
     {{ govukRadios({
       fieldset: {
         legend: {
-          html: "Choose response",
+          html: "Choose response (you'll be able to add conditions and recommendations)",
```

Comment:
I'm not sure about this ‘You'll be able to...’ text at all. Where has this line come from?

If we are going to have it then I think this should be hint text.